tests/test_api.py

```diff
@@ -43,6 +47,10 @@ def test_status_codes(self, client, endpoint, method, status_code):
         ('/diagnoses/123', 'GET', 'could not find diagnosis `123`'),
         ('/diagnoses/123', 'PUT', 'could not find diagnosis `123`'),
         ('/diagnoses/123', 'DELETE', 'could not find diagnosis `123`'),
+        ('/demographics', 'GET', 'success'),
```

Comment:
... I know I'm terrible...I snuck in a couple missing tests for a couple entities